NebulaModel/MultiplayerOptions.cs

```diff
@@ -0,0 +1,17 @@
+﻿using NebulaModel.Attributes;
+using System;
+
+namespace NebulaModel
+{
+    [System.Serializable]
+    public class MultiplayerOptions : ICloneable
+    {
+        [UIControl("Host Port")]
+        public int HostPort { get; set; } = 8469;
```

Comment:
As @TheRealMarine mentioned, this should be an ushort, with `Range(1, ushort.MaxValue)`